kubectl-plugin/test/e2e/testdata/ray-job.interactive-mode.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: ray.io/v1
+kind: RayJob
+metadata:
+  name: rayjob-sample
+spec:
+  # The current value is "InteractiveMode", meaning that it will wait for user to submit job and provide the job submission ID
+  submissionMode: 'InteractiveMode'
+  runtimeEnvYAML: |
+    pip:
+      - requests==2.26.0
+      - pendulum==2.1.2
```

Comment:
`requests` is one of the dependency of `ray`, so we'd better use other packages. For example, `emoji`.

Reply:
Decided to use `emoji` and `pyjokes`.